### DISCUSSION TITLE
What should we use to build docs.opensauced.pizza

### DISCUSSION THREAD

**Discussion by bdougie** [open-sauced/open-sauced#1039] [Ideas] [answered]
Thanks to @mtfoley, we now have our first fully thought-through docs pages and I plan on more. 

https://github.com/open-sauced/open-sauced/issues/1035

**Comment by bdougie** [comment]
The thought is to render this [folder](https://github.com/open-sauced/open-sauced/tree/main/docs) to a docs.opensauced.pizza site.  

anyone have experience with https://docusaurus.io/

**Comment by 0-vortex** [comment]
Did some research on [docz](https://www.docz.site) last week, it's down to docz vs docusaurus here as well. We can make this an issue and I can `.take` it IMO

**Comment by filiptronicek** [comment, marked as answer]
I would say Docosaurus would be a great option! I have seen many libraries use it and it was always a delightful experience for me.

**Reply by 0-vortex** [reply]
check out the pull request, could always improve before merging :D
